installing/installing_gcp_user_infra/installing-gcp-user-infra.adoc

```diff
@@ -0,0 +1,84 @@
+[id="installing-gcp-user-infra"]
+= Installing a cluster on GCP using Deployment Manager templates
+include::modules/common-attributes.adoc[]
+:context: installing-gcp-user-infra
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on
+Google Cloud Platform (GCP) using infrastructure that you provide.
+
+The steps for performing a user-provided infrastructure install are outlined here. Several
+link:https://cloud.google.com/deployment-manager/docs[Deployment Manager] templates are provided to assist in
+completing these steps or to help model your own. You are also free to create
+the required resources through other methods; the templates are just an
+example.
+
+[id="installation-gcp-user-infra-config-project"]
```

Comment:
I'd make the account creation assembly a prereq.

Reply:
N/A after discussion.